src/servers/src/mysql/handler.rs

```diff
@@ -237,7 +255,11 @@ impl MysqlInstanceShim {
                     schema: None,
                 },
                 stmt_key,
-            );
+            )
+            .map_err(|e| {
+                warn!("Failed to save prepared statement: {}", e);
```

Comment:
We can log errors in this style.
```suggestion
                warn!(e; "Failed to save prepared statement");
```